FirebaseMessaging/Sources/FIRMessaging.m

```diff
@@ -22,10 +22,10 @@
 #import <FirebaseInstanceID/FirebaseInstanceID.h>
 #import <FirebaseMessaging/FIRMessaging.h>
 #import <FirebaseMessaging/FIRMessagingExtensionHelper.h>
-#import <GoogleUtilities/GULAppDelegateSwizzler.h>
```

Comment:
Hey @paulb777 :-), turns out react-native-firebase was using the GULAppDelegateSwizzler:

https://github.com/invertase/react-native-firebase/blob/67c52a8aaeac32d2253e73769313e49700bc497c/packages/messaging/ios/RNFBMessaging/RNFBMessaging%2BAppDelegate.m#L20

And with 6.7.0 of GoogleUtilities / 6.28.0 of firebase-ios-sdk we get a build error https://github.com/invertase/react-native-firebase/issues/3938 as GULAppDelegateSwizzler.h tries to pull in GULApplication.h but it's not found:

> ios/Pods/Headers/Private/GoogleUtilities/GULAppDelegateSwizzler.h:19:9: 'GoogleUtilities/AppDelegateSwizzler/Private/GULApplication.h' file not found

I'm ace at the Java side of the react-native house but my Objective-C skills sadly leave something to be desired.

I attempted to alter the import in our repo to match the change here (` #import "GoogleUtilities/AppDelegateSwizzler/Private/GULAppDelegateSwizzler.h"`) without success

Is there a simple trick I'm missing that will resolve the headers and allow our auto-swizzling for react-native library consumers to work, or is this a more serious structural issue?

Sorry to bother, but thanks in advance for any help

Reply:
Turns out that the Facebook SDK for Unity was also using GULAppDelegateSwizzler. :/. We temporarily fixed the issue by adding this entry to the Podfile on our project:

`pod 'GoogleUtilities', '6.6.0'`

Hope this is fixed soon.

Reply:
At least our repo isn't alone @alejobrainz :-), we have a user report confirming that pinning the GoogleUtilities dep at 6.6.0 works for us as well, at least for firebase-ios-sdk pods 6.27.1, still building against 6.28.0 but with GoogleUtilities pinned on 6.6.0 to see

Reply:
Sorry for the issue - I see it's been reported at #6047 so we'll make all progress updates there and continue the discussion for greater visibility.